### PR TITLE
update Spanish translation

### PR DESCRIPTION
**Pull request by ancouli**
The template file in Transifex was updated. I am not sure if the update in the first line is correct. In Transifex translators are asked to translate "Deutsch" but in every other file it was translated with the name of the target language.

### CODE REVIEW

**Comment by xi** [comment]
> In Transifex translators are asked to translate "Deutsch" but in every other file it was translated with the name of the target language.

Yes, that is an unfortunate effect: This field is supposed to be the name of the language, but transifex only displays the translation in the "original" language.

File: langs/es.ini [hunk 1]
```diff
@@ -1,11 +1,19 @@
-name="Español"
+name="Alemán"
 dir="ltr"
-missing="Lamentableme no hay traduccion disponible"
+missing="Lamentablemente no hay ninguna traducción disponible"
 readmore="Leer más"
+languages="Idiomas"
+section="Sección"
+shortcuts="Abreviaturas"
+legal="legal"
```
Comment (xi) [inline langs/es.ini:8]: Is this an actual translation or has transifex just copied the original string?

File: langs/es.ini [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
 email="Correo electrónico"
 fax="Fax"
 telephone="Teléfono"
 bus="Autobús"
 subway="Metro"
+skip="Ir al contenido"
+newsletter="Suscribirse al boletín de noticias"
```
Comment (xi) [inline langs/es.ini:15]: This doesn't look right.

File: langs/es.ini (hunk 1 above)
Comment (ancouli) [reply langs/es.ini:8]: This is an actual translation.

File: langs/es.ini (hunk 2 above)
Comment (ancouli) [reply langs/es.ini:15]: Why? It corresponds to the German source "Zum Inhalt springen" and "Newsletter abonnieren"

Comment (xi) [reply langs/es.ini:15]: Ah, you are right, sorry. I thought had ought to be a translation of "newsletter". My mistake.

**Comment by ancouli** [comment]
> > In Transifex translators are asked to translate "Deutsch" but in every other file it was translated with the name of the target language.
> 
> Yes, that is an unfortunate effect: This field is supposed to be the name of the language, but transifex only displays the translation in the "original" language.

Okay. I will correct it in transifex and update the file.

**Comment by xi** [comment]
thanks :tada:

**Pull request merged**